controllers/auth.controller.js

```diff
@@ -53,7 +53,10 @@ exports.signin = async (req,res)=>{
             });
         }
 
-        const token = jwt.sign({id: user.userId}, authConfig.secret, {expiresIn : process.env.JWT_TIME});
+        const accessToken = jwt.sign({id: user.userId}, authConfig.secret, {expiresIn : process.env.JWT_TIME});
```

Comment:
Avoid reading with process in this file. A better place could be the config file